lifecycle/staging/src/github.com/labring/image-cri-shim/.github/workflows/close-pr.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright © 2025 sealos.
```

Comment:
How does this workflow have to do with image-cri-shim cache feature?

Reply:
> How does this workflow have to do with image-cri-shim cache feature?

It's fine, there's an LIC check.